components/emoji_picker/emoji_picker_overlay.jsx

```diff
@@ -59,6 +59,16 @@ export default class EmojiPickerOverlay extends React.PureComponent {
     }
 
     render() {
+        const emojiTrigger = this.props.target();
+        let rightOffset = Constants.DEFAULT_EMOJI_PICKER_RIGHT_OFFSET;
+        if (emojiTrigger) {
+            rightOffset = window.innerWidth - emojiTrigger.getBoundingClientRect().left - Constants.DEFAULT_EMOJI_PICKER_LEFT_OFFSET;
+
+            if (rightOffset < 0) {
```

Comment:
Should we compare against `Constants.DEFAULT_EMOJI_PICKER_RIGHT_OFFSET`? Otherwise the transition across this edge might look like:
```
5, 4, 3, 2, 1, 0, 15
```

Reply:
Nice call. 👍